docs/lightning/thunderhub.md

```diff
@@ -1,5 +1,44 @@
+
 # Thunderhub
 
-GUI to manage lightning channels
+A GUI to manage lightning channels
+
+### Introduction
+
+ThunderHub is an open-source LND node manager where you can manage and monitor your node on any device or browser. It allows you to take control of the lightning network with a simple and intuitive UX.
+
+ThunderHub is a creation of Anthony Potdevin, and complete information can be found at either [Github](https://github.com/apotdevin/thunderhub) or the [ThunderHub
+website](https://thunderhub.io).
+
+![ThunderHub GUI](/docs/.vuepress/public/images/lightning/ThunderHub/ThunderHub_GUI_comp.png "ThunderHub GUI")
+
+### Features
+
+- Monitoring
+- Overview of current and pending balance for the Lightning and Bitcoin wallets.
+- Liquidity report with total remote and local lightning balance.
+- Complete network info.
+- View open/pending/closed channels and how balanced they are.
+- View all transactions.
+- View all forwarded payments.
+- View all chain transactions.
+- View all unspent UTXOS.
+
+### Management
+
+- Send and Receive Lightning payments.
+- Send and Receive Bitcoin payments.
+- Open and close channels.
+- Balance your channels through circular payments. (Check out the Tutorial)
+- Backup, verify and recover all your channels.
+- Sign and verify messages.
+
+## ThunderHub on myNode
+
+To use ThunderHub on myNode, open the main myNode page on a browser and find the ThunderHub emblem.
+
+![ThunderHub Emblem](/docs/.vuepress/public/images/lightning/ThunderHub/ThunderHub_emblem_comp.png "ThunderHub Emblem")
 
```

Comment:
probably you wanted to have some other image here, but this is same as the last one